mantidproject/mantid
# Rename ALFView files and restructure

**Description of work.**
The file and varaible names for ALF view were not very good. This PR improves them, they now follow the coding standards. I have also moved the common functionality to a more readily available part of Mantid. 


**To test:**
Open Mantid
Go to custom interfaces -> Direct -> ALF view
An interface should apper
In the run box type `80511` and press enter
Wait a little while and an updated instrument should appear in the GUI
Go to the pick tab
Select the `tube` option
Change the subplot from the pick tab to integrate (right click)
Change the axis to `out of plane angle` (right click)
On the instrument view, right click above a bright spot and select `export tube`
A plot should appear on the far right of the GUI
You should be able to press the fit button and it will do a fit.




Fixes #26904. 



Does not need to be in release notes, this is a maintenance task, 
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
